Add "Desktop" under computer and tech guidelines

danielsischy opened the pull request:
<!-- Thank you for contributing a pull request! Here are a few tips to help you:

1. Learn how to update the handbook: https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook
2. If this is your first contribution, please sign the Contributor License Agreement: http://www.mattermost.org/mattermost-contributor-agreement/
   - Once signed, you will be added to the Mattermost Approved Contributor List: https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true
   - If you included your mailing address, you may receive a Limited Edition Mattermost Mug as a thank you gift after your pull request is merged: https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143
-->

#### Summary
<!--
A description of what this pull request does.
-->

danielsischy committed:
Add "Desktop" under computer and tech guidelines

mattermod commented:
Hello @danielsischy,

Thanks for your pull request! A Core Committer will review your pull request soon. For code contributions, you can learn more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review/).

Per the [Mattermost Contribution Guide](https://developers.mattermost.com/contribute/getting-started/), we need to add you to the list of [approved contributors](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true) for the Mattermost project.

**Please help complete the Mattermost [contribution license agreement](http://www.mattermost.org/mattermost-contributor-agreement/)?**
Once you have signed the CLA, please comment with `/check-cla` and confirm that the CLA check is green.

This is a standard procedure for many open source projects.

Please let us know if you have any questions.

We are very happy to have you join our growing community! If you're not yet a member, please consider joining our [Contributors community channel](https://pre-release.mattermost.com/core/channels/tickets) to meet other contributors and discuss new opportunities with the core team.

TQuock committed:
Update README.md

Update documentation for content consistency and tone of the how to spend company money policy

amyblais commented on company/how-to-guides-for-staff/how-to-spend-company-money/README.md:
```diff
@@ -72,6 +72,35 @@ For all new staff members, to request a new MacBook, Lenovo, or other laptop, pl
 
+#### Desktop \(Business computer\)
+
+Desktops may be requested alongside a laptop for **engineering** roles. To request a desktop, email a request to your manager with the subject `Desktop for Approval`. Include IT for approval tracking by cc'ing daniel.sischy@mattermost.com. If your manager approves, a request should be sent to Chris Overton, VP of Engineering, for budget review and need. If you receive both approvals, you may form ...
```
```suggestion
Desktops may be requested alongside a laptop for **engineering** roles. To request a desktop, email a request to your manager with the subject `Desktop for Approval`. Include IT for approval tracking by cc'ing daniel.sischy@mattermost.com. If your manager approves, a request should be sent to Chris Overton, VP of Engineering, for budget review and need. If you receive both approvals, you may formally request a desktop through the [Desktop Request Form](https://docs.google.com/forms/d/e/1FAIpQLSe2_21Jllo8kztsnhhF_Xre9iOiPoeELUWwYziCDB25y68Wpw/viewform). Estimated delivery time will be communicated by IT once the order has been placed, but expect on average 1-4 week delivery window. 
```

amyblais commented on company/how-to-guides-for-staff/how-to-spend-company-money/README.md:
```diff
@@ -72,6 +72,35 @@ For all new staff members, to request a new MacBook, Lenovo, or other laptop, pl
+
+Desktops may be requested alongside a laptop for **engineering** roles. To request a desktop, email a request to your manager with the subject `Desktop for Approval`. Include IT for approval tracking by cc'ing daniel.sischy@mattermost.com. If your manager approves, a request should be sent to Chris Overton, VP of Engineering, for budget review and need. If you receive both approvals, you may form ...
+
+We offer three desktop options. The specifications below are those believe needed for you to do your best work at Mattermost, but are not a hard ceiling. If you need upgraded specifications, outline them within the [Desktop Request Form](https://docs.google.com/forms/d/e/1FAIpQLSe2_21Jllo8kztsnhhF_Xre9iOiPoeELUWwYziCDB25y68Wpw/viewform). 
```
```suggestion
We offer three desktop options. The specifications below are those we believe are needed for you to do your best work at Mattermost, but they are not a hard ceiling. If you need upgraded specifications, outline them within the [Desktop Request Form](https://docs.google.com/forms/d/e/1FAIpQLSe2_21Jllo8kztsnhhF_Xre9iOiPoeELUWwYziCDB25y68Wpw/viewform).
```

danielsischy committed:
Update company/how-to-guides-for-staff/how-to-spend-company-money/README.md

Co-authored-by: Amy Blais <amy_blais@hotmail.com>

danielsischy committed:
Update company/how-to-guides-for-staff/how-to-spend-company-money/README.md

Co-authored-by: Amy Blais <amy_blais@hotmail.com>
